config/routes.rb

```diff
@@ -485,6 +485,22 @@
   end
   ####
 
+  #### AdminNote controller
```

Comment:
be nicer to have shallow routes for new so that we can do e.g. `/admin/bodies/3/notes/new` (which automatically sets `notable_type` & `notable_id`), but then editing the created note is simply `/admin/note/:id/edit`. not essential this minute though